game-of-life-multiplatform/desktop-jvm/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'kotlin'
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    compile project(":common-jvm")
```

Comment:
Nit: could you use single quotes where we don't need interpolation please? (here and below)